### PR TITLE
[Solver docs] Add CIP-36 updates

### PR DESCRIPTION
**Pull request by harisang**
This PR updates the solver rewards part with the latest [CIP-36](https://snapshot.org/#/cow.eth/proposal/0x4e58f9c1208121c0e06282b5541b458bc8c8b76090263e25448848f3194df986) changes, that just passed.

### CODE REVIEW

File: docs/cow-protocol/reference/core/auctions/rewards.md [hunk 1]
```diff
@@ -89,17 +95,19 @@ $$
...
+
+As specified in [CIP-27](https://snapshot.org/#/cow.eth/proposal/0x64e061568e86e8d2eec344d4a892e4126172b992cabe59a0b24c51c4c7e6cc33) and [CIP-36](https://snapshot.org/#/cow.eth/proposal/0x4e58f9c1208121c0e06282b5541b458bc8c8b76090263e25448848f3194df986), solvers that participate in the price estimation competition get rewarded as follows. Each order that is within market price, is associated with a quote that was computed as part of the pric eestimation competition, and was used in order to compute the limit price of the order. The protocol keeps track of the quote associated with each created order and the corresponding solver that provided the quote, and if and when the order gets executed, the solver that provided the quote (that can be different compared to the solver that ended up executing the order) gets rewarded with $$min{0.0006 ETH, 6 COW}$$, where again, the conversion from ETH to COW is done by using an up-to-date price.
```
Comment (mfw78) [inline docs/cow-protocol/reference/core/auctions/rewards.md]:
```suggestion
As specified in [CIP-27](https://snapshot.org/#/cow.eth/proposal/0x64e061568e86e8d2eec344d4a892e4126172b992cabe59a0b24c51c4c7e6cc33) and [CIP-36](https://snapshot.org/#/cow.eth/proposal/0x4e58f9c1208121c0e06282b5541b458bc8c8b76090263e25448848f3194df986), solvers that participate in the price estimation competition get rewarded as follows. Each order that is within market price, is associated with a quote that was computed as part of the price estimation competition, and was used in order to compute the limit price of the order. The protocol keeps track of the quote associated with each created order and the corresponding solver that provided the quote, and if and when the order gets executed, the solver that provided the quote (that can be different compared to the solver that ended up executing the order) gets rewarded with $$min{0.0006 ETH, 6 COW}$$, where again, the conversion from ETH to COW is done by using an up-to-date price.
```

Comment (mfw78) [reply docs/cow-protocol/reference/core/auctions/rewards.md]: Typo

**Review by mfw78 — CHANGES_REQUESTED** [review]

**Review by mfw78 — APPROVED** [review]

**Review by fhenneke — APPROVED** [review]

**Comment by gogonimago** [comment]
Ok I've reviewed, the grammar looks good, the one comment I have is I would remove the punctuation and spacing that occurs after variable names and equations in several places. I would simply continue the previous sentence with no punctuation or spacing or start a new sentence under each instance of this.

Screenshot 2024-02-29 at 2 20 57 PM

**Comment by mfw78** [comment]
@gogonimago Would you be able to file a supplemental PR for this? :pray:

**Pull request merged**